files/setup.sh

```diff
@@ -55,20 +55,30 @@ else
 
     if [ -n "${HTTP_PROXY}" ] || [ -n "${HTTPS_PROXY}" ]; then
         PROXY_CONF=/etc/sysconfig/proxy
+        mkdir -p /etc/systemd/system/docker.service.d
+        DOCKER_PROXY=/etc/systemd/system/docker.service.d
```

Comment:
You should define `DOCKER_PROXY` first and then do `mkdir -p ${DOCKER_PROXY}`

Reply:
- [x] Done Thx @lamw